docs/minimize_energy/dwave.embedding.chain_breaks.MinimizeEnergy.rst

```diff
@@ -5,12 +5,18 @@
 
 .. autoclass:: MinimizeEnergy
 
-
+   
    .. automethod:: __init__
 
-
+   
    .. rubric:: Methods
 
    .. autosummary::
-
+   
       ~MinimizeEnergy.__init__
+   
+   
+
+   
+   
+   
```

Comment:
A lot of unneeded whitespace here. Is this file auto-generated? Can the whitespaces be removed somehow?

Reply:
Yes, I've removed the white spaces by deleting the file. It's an autogenerated file that accidently got added to the PR